src/wp-admin/includes/misc.php

```diff
@@ -1398,6 +1398,15 @@ function wp_admin_canonical_url() {
 	// Ensure we're using an absolute URL.
 	$current_url  = set_url_scheme( 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'] );
 	$filtered_url = remove_query_arg( $removable_query_args, $current_url );
+
+	/**
+	 * Filters the admin canonical url value
```

Comment:
Missing final period in this sentence :)

Reply:
thank you! Updated PR